QEfficient/utils/_utils.py

```diff
@@ -93,7 +93,18 @@ def qpc_exists(qpc_dir_path: str) -> bool:
     return qpc_exists_bool
 
 
-def onnx_exists(model_name: str) -> Tuple[bool, str, str]:
+def get_onnx_dir_name(model_name, has_fbs):
+    # Create a unique directory name for the ONNX model
+    # Clearly indicate whether it's with or without FBS
+    # Replace all hyphens with underscores
+    model_name_safe = model_name.replace("/", "_").replace("-", "_")
+    if has_fbs:
+        return f"onnx_{model_name_safe}_with_fbs"
```

Comment:
should this also save info about value of fps in the name?

Reply:
No. Model should be same for any value of FBS. Its a dynamic variable input that we can provide through specialization file.